hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MemStoreCompactor.java

```diff
@@ -160,8 +163,9 @@ private boolean shouldFlatten() {
     // limit the number of the segments in the pipeline
     int numOfSegments = versionedList.getNumOfSegments();
     if (numOfSegments > 3) {        // hard-coded for now as it is going to move to policy
-      LOG.debug("In-Memory shrink is doing compaction, as there already are " + numOfSegments
+      LOG.debug("In-Memory shrink is doing MERGE, as there already are " + numOfSegments
```

Comment:
Same here, too many concepts - shrink, merge, compact. Can we just say "index compaction triggered".